privet/avahi.go

```diff
@@ -4,11 +4,12 @@
 // license that can be found in the LICENSE file or at
 // https://developers.google.com/open-source/licenses/bsd
 
-// +build linux
+// +build linux freebsd
 
 package privet
 
-// #cgo LDFLAGS: -lavahi-client -lavahi-common
+// #cgo CFLAGS: -I%%PREFIX%%/include
```

Comment:
I told you before that I would try this out on Linux and OSX, and I did not. I'll try this patch on those platforms now and comment here again.